packages/admin/dashboard/src/i18n/translations/__tests__/validate-translations.spec.ts

```diff
@@ -38,29 +39,36 @@ function getTranslationKeys(obj: any, prefix = ""): string[] {
   return keys.sort()
 }
 
-describe("translation schema validation", () => {
-  test("en.json should have all keys defined in schema", () => {
-    const enPath = path.join(translationsDir, "en.json")
-    const enTranslations = JSON.parse(fs.readFileSync(enPath, "utf-8"))
+function validateTranslationFile(fileName: string) {
```

Comment:
Hi @DanSilva41, please undo the changes to the test file. We can't maintain translations in other languages, and this would prevent us from merging any PR's that changes the i18n schema, without also updating all the language files, which we can't.

Reply:
Understood, it makes sense, @kasperkristensen.

I'll undo.

Reply:
It's done.

Thanks, @kasperkristensen!